rhel/SPECS/dbaas-tools.spec

```diff
@@ -0,0 +1,69 @@
+%define debug_package %{nil}
+%global provider_aws        github
+%global provider_tld_aws    com
+%global project_aws         kubernetes-sigs
+%global repo_aws            aws-iam-authenticator
+%global provider_prefix_aws %{provider_aws}.%{provider_tld_aws}/%{project_aws}/%{repo_aws}
+
+# the lines below are sed'ed by build-server-rpm script to set a correct version
```

Comment:
I'm pretty sure they are not

Reply:
Removed in 
98ed9a4 

They were changed by these:
```
        if [[ ${spec_name} = "dbaas-tools" ]]; then
            sed -i -e "s/global commit_aws.*/global commit_aws ${full_commit}/" ${rpmbuild_dir}/SOURCES/${spec_name}.spec
        fi
```